main_2023.py

```diff
@@ -1,30 +1,55 @@
 """
 For 2022-2023 UAS competition.
 """
+import argparse
 import multiprocessing as mp
+from pathlib import Path
 
 import cv2
+import yaml
 
 from utilities.workers import queue_proxy_wrapper
 from utilities.workers import worker_controller
 from utilities.workers import worker_manager
 from modules.detect_target import detect_target_worker
 from modules.video_input import video_input_worker
 
+CONFIG_FILE_PATH = Path("./config.yaml")
 
-QUEUE_MAX_SIZE = 10
-
-VIDEO_INPUT_CAMERA_NAME = 0
-VIDEO_INPUT_WORKER_PERIOD = 1.0  # seconds
-VIDEO_INPUT_SAVE_PREFIX = "log_image"
-
-DETECT_TARGET_WORKER_COUNT = 1
-DETECT_TARGET_DEVICE = 0  # Use "cpu" if no CUDA
-DETECT_TARGET_MODEL_PATH = "tests/model_example/yolov8s_ultralytics_pretrained_default.pt"  # TODO: Update
-DETECT_TARGET_SAVE_PREFIX = "log_comp"
+if __name__ == "__main__":
```

Comment:
With all of these try-except blocks, it's probably worth making this a function and then calling it from the entry:
```py
def main() -> int:
    ...
    return 0

if __name__ == "__main__":
    result_run = run()
    if result_run < 0:
        print(f"ERROR: Status code: {result_run}")

    print("Done!")
```